sphinx/source/members/index.rst

```diff
@@ -0,0 +1,30 @@
+Member Governance
+=================
+
+This section describes how a consortium of trusted :term:`members` governs an existing CCF network. It explains how members can submit proposals to CCF and how these proposals are accepted based on the rules defined in the :term:`constitution`.
+
+Before creating a new CCF network (see :ref:`Starting a New Network`), the identity of the initial member(s) of the consortium must be generated.
```

Comment:
I think we can have those as just links (as opposed to (see: ...)), but we should keep the cross referencing and allow users to jump in rather than force them to browse through the hierarchy and find their way to the point of detail they want to delve into.